Sources/MlemMiddleware/Content Models/User/UserStub.swift

```diff
@@ -103,9 +100,30 @@ public final class UserStub: UserProviding, Codable {
         try container.encode(lastLoggedIn, forKey: .lastUsed)
         try container.encode(api.baseUrl, forKey: .instanceLink)
     }
+    
+    public var keychainId: String {
+        getKeychainId(actorId: actorId)
+    }
```

Comment:
This feels redundant with `getKeychainId` itself--can we just define this as `"\(actorId.absoluteString)_accessToken"`? We can then do the same thing and define a `legacyKeychainId` that performs the int-based keychain id creation

Reply:
`getKeychainId` needs to exist outside of the class because it is called in the `init(from decoder: Decoder)` initialiser - if it were inside the struct, it would be un-callable until all properties are initialised, which isn't possible.

Reply:
Makes sense 👍